docs/intro_testers.rst

```diff
@@ -0,0 +1,61 @@
+.. _intro_testers:
+
+========================
+Introduction for Testers
+========================
+
+This document is aimed at providing an introduction to running existing test suites.
+
+We assume here that you have access to an operational test lab; if not, ask
+your local admin for access!
+
+If you're here to test upstream Ceph, start `here
+<http://ceph.github.io/sepia/adding_users/>`__.
+
+
+Terminology
+===========
+
+In the abstract, each set of tests is defined by a `suite`. All of our suites
+live in the `ceph-qa-suite repository
+<https://github.com/ceph/ceph-qa-suite/>`__, in the `suites` subdirectory.
+
+In concrete terms, a `run` is what is created by assembling the contents of a
+`suite` into a number of `jobs`. A `job` is created by assembling a number of
+`fragments` (also known as `facets`) together. Each `fragment` is in `YAML
+<http://yaml.org/>`__ format.
+
+Each `job` definition contains a list of `tasks` to execute, along with
+`roles`. `Roles` tell `teuthology` how many nodes to use for each `job` along
+with what functions each node will perform.
+
+To go into more depth regarding suite design, see `ceph-qa-suite`'s `README`.
+
+One example of this is the `smoke
+<https://github.com/ceph/ceph-qa-suite/tree/master/suites/smoke>`__ suite.
+
+
+Scheduling
+==========
+Most testing happens by scheduling `runs`. The way we do that is using the
+`teuthology-suite` command.
+
+To get a preview of what `teuthology-suite` might do, try::
+
+    teuthology-suite -v -s smoke -c master -m mira --dry-run
+
```

Comment:
We should probably explain here that the -m mira option in this example specifies name of the set of machines from which teuthology-suite is selecting.